web/src/engine/websites/MangaDex.ts

```diff
@@ -40,8 +40,8 @@ type APIChapter = {
     relationships: {
         id: string
         type: string
-        attributes: {
-            name: string
+        attributes?: {
+            name?: string
```

Comment:
Both can be `null` attribute and name, or is only attribute nullable?

Reply:
IIRC "attributes" was missing due to a DB error.

> "Cannot read properties of undefined (reading 'name')"